modules/influxdb/collector.go

```diff
@@ -0,0 +1,44 @@
+package influxdb
+
+import (
+	"github.com/NETWAYS/support-collector/pkg/collection"
+	"os"
+)
+
+const ModuleName = "influxdb"
+
+var relevantPaths = []string{
+	"/etc/influxdb",
+	"/var/lib/influxdb",
+}
+
+var files = []string{
+	"/etc/influxdb/config.*",
```

Comment:
Standard config file would be `/etc/influxdb/influxdb.conf` AFAIK, so why not:

```suggestion
	"/etc/influxdb",
```